config/symbols.pspeu.stst0.txt

```diff
@@ -164,6 +164,19 @@ g_eDamageDisplayClut = 0x09279978;
 eRoomForegroundInit = 0x092799B0;
 g_isSecretStairsButtonPressed = 0x09279D98;
 D_80180908 = 0x09279DA0;
+D_psp_E_DRACULA_UNK2E = 0x09279DC8;
+D_psp_E_ID_2D = 0x09279DD0;
+D_psp_E_DRACULA_UNK2C = 0x09279DD8;
+D_psp_E_DRACULA_UNK2B = 0x09279DE0;
+D_psp_E_DRACULA_UNK23 = 0x09279E20;
+D_psp_E_DRACULA_FIREBALL = 0x09279E28;
+D_psp_E_DRACULA_UNK21 = 0x09279E30;
+D_psp_E_DRACULA_UNK20 = 0x09279E38;
+D_psp_E_DRACULA_UNK1F = 0x09279E40;
+D_psp_E_DRACULA_METEOR = 0x09279E48;
+D_psp_E_DRACULA_UNK1D = 0x09279E50;
+D_psp_E_DRACULA_UNK1C = 0x09279E58;
+D_psp_E_DRACULA_UNK1B = 0x09279E60;
```

Comment:
These files are not yet imported for PSP, but once done these mappings can be removed